test-cases/longevity/longevity-lwt-24h-multidc.yaml

```diff
@@ -0,0 +1,30 @@
+test_duration: 1560
+prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic_multidc.yaml n=10000000 ops'(insert=1)' cl=QUORUM -mode native cql3 -rate threads=50" ]
+stress_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic_multidc.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1)' cl=QUORUM duration=1440m -mode native cql3 -rate threads=30" ]
+stress_read_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic_multidc.yaml ops'(select=1)' cl=SERIAL duration=1440m -mode native cql3 -rate threads=30" ]
+
+n_db_nodes: '4 3 2'
+n_loaders: 3
+n_monitor_nodes: 1
+experimental: true
```

Comment:
It's not experimental any more. We can remove the flag

Reply:
@amoskong ping

Reply:
Fixed, thanks.